src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2989,7 +2936,7 @@ bool AdaptDispatch::DesignateCodingSystem(const VTID codingSystem)
 // - charset - The identifier indicating the charset that will be used.
 // Return value:
 // True if handled successfully. False otherwise.
-bool AdaptDispatch::Designate94Charset(const VTInt gsetNumber, const VTID charset)
+void AdaptDispatch::Designate94Charset(const VTInt gsetNumber, const VTID charset)
 {
     return _termOutput.Designate94Charset(gsetNumber, charset);
 }
```

Comment:
There's another half dozen `return` statements here that can assumedly be removed.